lib/urldata.h

```diff
@@ -519,7 +519,7 @@ struct Curl_async {
   int port;
   struct Curl_dns_entry *dns;
   int status; /* if done is TRUE, this is the status from the callback */
-  void *os_specific;  /* 'struct thread_data' for Windows */
+  struct thread_data *tdata; /* 'struct thread_data' for Windows */
```

Comment:
```suggestion
  struct thread_data *tdata;
```
outdated comment, otherwise looks ok

Reply:
Ack, will amend before merge!